docker/test/integration/minifi/core/DockerTestCluster.py

```diff
@@ -110,6 +111,12 @@ def check_s3_server_object_metadata(self, content_type="application/octet-stream
         server_metadata = json.loads(metadata_json)
         return server_metadata["contentType"] == content_type and metadata == server_metadata["userMetadata"]
 
+    @retry_check()
+    def check_azure_storage_server_data(self, test_data):
+        data_file = subprocess.check_output(["docker", "exec", "azure-storage-server", "find", "/data/__blobstorage__", "-type", "f"]).decode(sys.stdout.encoding).strip()
```

Comment:
`sys.stdout.encoding` is not GitHub Actions safe. See #1018 .
![image](https://user-images.githubusercontent.com/64011968/109508009-a74a8680-7a9f-11eb-92df-94101ba09a74.png)

Reply:
Updated in [48ec7a](https://github.com/apache/nifi-minifi-cpp/pull/979/commits/48ec7a2ebb6d109c147f2a3b83aa2ae8996acfa2). It may cause a merge conflict, but I will solve it if it occurs.

Reply:
Hope I can merge first :D